From d23d21bc76693dd2a43b5d76c68972ac01a055ba Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <gentoo@mgorny.alt.pl>
Date: Sat, 20 Jul 2013 15:56:56 +0200
Subject: [PATCH] Remove insecure RUNPATHs.

LLVM seems to build fine without them, and they were removed for 3.4
already.
---
 Makefile.rules | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Makefile.rules b/Makefile.rules
index db252f7..a29481b 100644
--- a/Makefile.rules
+++ b/Makefile.rules
@@ -639,11 +639,6 @@ ifneq ($(HOST_OS), $(filter $(HOST_OS), Cygwin MingW))
   ifneq ($(HOST_OS), Darwin)
     ifdef TOOLNAME
       LD.Flags += $(RPATH) -Wl,'$(PROJ_libdir)'
-      ifdef EXAMPLE_TOOL
-        LD.Flags += $(RPATH) -Wl,$(ExmplDir) $(DynamicFlag)
-      else
-        LD.Flags += $(RPATH) -Wl,$(ToolDir) $(DynamicFlag)
-    endif
   endif
 else
   ifneq ($(DARWIN_MAJVERS),4)
-- 
1.8.3.2

